stable/spinnaker/templates/statefulsets/halyard.yaml

```diff
@@ -13,6 +13,8 @@ spec:
       component: halyard
   template:
     metadata:
+      annotations:
```

Comment:
Can you wrap this with an if statement to check if the .Values.halyard.podAnnotations has items or is provided so it will print it please?

Reply:
Also can you add an empty podAnnotations: or some annotations to the values.yaml file please

Reply:
@rverma-nikiai

Reply:
@dwardu89 Since this was already merged limiting this to just adding custom service profiles.